pkg/s3wrapper/client.go

```diff
@@ -138,14 +141,19 @@ func (c *S3Client) CreateBucket() error {
 	return nil
 }
 
-func (c *S3Client) UploadStream(ctx context.Context, reader io.Reader, objectName string) error {
+func (c *S3Client) UploadStream(ctx context.Context, reader io.Reader, objectName string, public bool) error {
 	log := logutil.FromContext(ctx, c.log)
 	uploader := s3manager.NewUploader(c.session)
-	_, err := uploader.Upload(&s3manager.UploadInput{
+	input := s3manager.UploadInput{
 		Bucket: aws.String(c.cfg.S3Bucket),
 		Key:    aws.String(objectName),
 		Body:   reader,
-	})
+	}
+	if public {
+		input.ACL = aws.String(s3.BucketCannedACLPublicRead)
```

Comment:
Will this hit the same issue as https://github.com/oshercc/ignition-manifests-and-kubeconfig-generate/pull/24 solved?

Reply:
I think we couldn't set the public read ACL in the app-sre managed S3 account

Reply:
Yes good call - we'll need to ask them about that.  Or maybe if we're in S3 we should just redirect to a URL like this rather than hosting it ourselves?  Need to see how to obtain that URL.
https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/rhcos-4.6.0-rc.4-x86_64-live-rootfs.x86_64.img

Reply:
Should we hold this PR on this? I think using the ACL previously failed the upload.

Reply:
Yeah, if those urls are consistent and reliable, that might be the better option.

Reply:
Added public bucket